bcolz/ctable.py

```diff
@@ -644,13 +644,17 @@ def copy(self, **kwargs):
         return ccopy
 
     @staticmethod
-    def fromdataframe(df, **kwargs):
+    def fromdataframe(df, index_as_columns=False, **kwargs):
```

Comment:
Although backward incompatible, I'd say that setting `index_as_columns` to `True` as default makes more sense.